sidebars.js

```diff
@@ -36,11 +36,8 @@ const sidebars = {
   ],
   tutorials: [
     {
-      type: 'category',
-      label: 'Tutorials',
-      link: { type: 'doc', id: 'tutorials/overview' },
-      collapsible: false,
-      items: [],
+      type: 'autogenerated',
+      dirName: 'tutorials',
```

Comment:
Is this used anymore?

Reply:
It's not but I left it in hope that it'd save someone else some confusion in the future. Happy to remove if you'd prefer

Reply:
You can leave it if it was left here intentionally.